### PR TITLE
feat: Added util function to generate unique ids

### PR DESCRIPTION
**Pull request by mfranceschit**
## Changes/solution

- Added util function to generate unique ids
It will be use for ticket EP-1881

### CODE REVIEW

File: src/utils/index.ts [hunk 1]
```diff
@@ -67,3 +68,11 @@ export const sortResourcesDependencies = (
     }
     return 0
   })
+
+/**
+ * Create an unique hash identifier
+ * @param entity entire entity to create identifier
+ * @returns unique identifier
+ */
+export const generateUniqueId = (entity: any): string =>
```
Comment (tyler-dunkel) [inline src/utils/index.ts:77]: Once we have this being used in one of the providers, can we do a performance test to understand what implications this has?

Comment (mfranceschit) [reply src/utils/index.ts:77]: Sure thing. My pilot will be AWS provider https://github.com/cloudgraphdev/cloudgraph-provider-aws/pull/83

Comment (tyler-dunkel) [reply src/utils/index.ts:77]: Sounds good, do you need an alpha release to be able to pull these into gql api or are you just going to locally link them?

Comment (mfranceschit) [reply src/utils/index.ts:77]: An alpha release would be great! I tested everything locally, and it doesn't seem to impact the performance significantly.

**Pull request merged**